projects/ui/src/lib/components/po-table/po-table-base.component.ts

```diff
@@ -640,4 +668,9 @@ export abstract class PoTableBaseComponent implements OnChanges {
     });
   }
 
+  /** Método que expande e colapsa o detalhe da linha. */
+  toggleDetail(row: any) {
+    this.setShowDetail(row, !row.$showDetail) ;
+  }
```

Comment:
Alvaro, por convenção agrupamos os métodos entre públicos, privados e protegidos. Poderia ajustar este com os demais por favor?

Reply:
Vou ajustar, mas estou com duvida quanto a ordenação dos métodos em ordem alfabética.
Nesse fonte eles não estão então eu coloquei ele acima do toggleRowAction pq se eu for ajustar todos os métodos que não tem haver com a minha PR acho que vai estragar o histórico do git. Concorda?